Makefile

```diff
@@ -4,15 +4,15 @@ PHONIES += jni-libs-no-unpack unpack unpack-clean
 
 GPG2      = gpg2
 JAVAH     = javah
-NDK_BUILD = ndk-build
+NDK_BUILD = /usr/local/Cellar/android-ndk/r12b/ndk-build
```

Comment:
Uhm, this should not be committed. At the same time, the current version is r13 and the code does not compile with r13: one of the architectures fail for unspecified reasons. We should merge this branch only after this issue has been solved, because of course hard-coding this path sucks way too much to land in master.